client/__tests__/e2e/e2e.test.ts

```diff
@@ -1722,7 +1879,8 @@ test("categories and values from dataset appear and properly truncate if applica
         .all();
 
       expect(Object.keys(categoryRows).length).toBe(1001);
-      await snapshotTestGraph(page, testInfo);
+
+      await takeSnapshot(page, getSnapshotPrefix(testInfo), testInfo);
```

Comment:
Why takeSnapshot and not snapshotTestGraph here?

Reply:
Oh I see you comment above - maybe different function names or a comment?

Reply:
Great idea! Adding the comment below thank you 😄 

```ts
/**
   * (thuang): Regular snapshot instead of `snapshotTestGraph`, since this test
   * doesn't involve the graph
   */
  await takeSnapshot(page, getSnapshotPrefix(testInfo), testInfo);
```